core/arch/arm/plat-stm32mp1/shared_resources.c

```diff
@@ -191,52 +191,6 @@ static void register_periph(enum stm32mp_shres id, enum shres_state state)
 	}
 
 	shres_state[id] = state;
-
-	/* Explore clock tree to lock secure clock dependencies */
```

Comment:
This should probably be part of another P-R, no?

Reply:
This change could be squashed in next commit "plat-stm32mp1: remove stm32mp_register_clock_parents_secure()". Would you prefer this way?